api/Element.json

```diff
@@ -730,6 +730,254 @@
           }
         }
       },
+      "animationcancel_event": {
+        "__compat": {
+          "description": "<code>animationcancel</code> event",
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Element/animationcancel_event",
+          "spec_url": "https://drafts.csswg.org/css-animations/#eventdef-animationevent-animationcancel",
```

Comment:
Can you link https://drafts.csswg.org/css-animations/#dom-document-onanimationcancel here and do similarly throughout?